en/orm/associations.rst

```diff
@@ -357,6 +357,23 @@ syntax::
         }
     }
 
+Sometimes you may want to configure composite keys in your associations::
+
+    $this->hasMany('Reviews', [
+        'foreignKey' => [
+            'article_id',
+            'article_name'
+        ],
+        'bindingKey' => [
+            'article_id',
+            'article_name'
+        ]
+    ]);
+
```

Comment:
Yes. The comment fits well. 👍 

If we avoid the auto setup of bindingKey in the example, I think we can show how to configure different keys with clarity.